test/module/irohad/torii/torii_mocks.hpp

```diff
@@ -24,6 +24,26 @@
 
 namespace iroha {
   namespace torii {
+
+    /**
+     * This function tries to get a specific error from the query respons
+     * might throw an exception !!!
```

Comment:
Could you please tell where an exception can be thrown? I checked boost::get() but it looks like it should do a compile time checks, isn't it?
Also I think this function should be moved from mocks - it's not a mock ;)